### PR TITLE
#571 - Added docs for path patterns support & fixed GET endpoint

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #571 
Added docs for path patterns support & fixed GET endpoint

### CODE REVIEW

File: REST_API.md [hunk 1]
```diff
@@ -143,6 +144,7 @@ Fields description:
 
 Field name | Type | Meaning | Required
 ------ | ------ | ------ | ------
+includesPattern | string | Path pattern to which permissions are applied | Y
```
Comment (olenagerasimova) [inline REST_API.md:147]: @olegmoz I'd suggest to simplify this sentence to "Path patterns to apply permissions to"

File: REST_API.md [hunk 2]
```diff
@@ -183,9 +186,15 @@ The following synonyms and shortened values for standard operations are supporte
 - `delete` - `d`
 - `manage` (any operation is allowed) - `m`, `admin`
 
+`include-patterns` is an optional argument (effectively is `["**"]` by default) 
+which allows specifying path patterns to which permissions are applied.
```
Comment (olenagerasimova) [inline REST_API.md:190]: @olegmoz I'd suggest to rephrase:
`include-patterns` is an optional argument (effectively value `["**"]` is set by default) 
which allows specifying path patterns to apply permissions to.

File: REST_API.md [hunk 3]
```diff
@@ -183,9 +186,15 @@ The following synonyms and shortened values for standard operations are supporte
...
+Path patterns are specified by Ant-like expressions.
+
 Possible responses:
 - `200 OK` when permissions were added successfully
 - `500 INTERNAL ERROR` in the case of unexpected server error
+- `400 BAD REQUEST` in the cases when repository name does not exist 
+ or `include-patterns` format is invalid.
```
Comment (olenagerasimova) [inline REST_API.md:197]: @olegmoz let's remove point at the end of the sentence, they are not required in the lists

File: src/test/java/com/artipie/api/artifactory/GetPermissionSliceTest.java [hunk 4]
```diff
@@ -129,25 +127,17 @@ void returnsUsersAndPermissionsList() {
...
-    ) {
-        final JsonArrayBuilder incpatterns = Json.createArrayBuilder();
-        for (final String pattern : patterns) {
-            incpatterns.add(pattern);
-        }
+    private byte[] response(final String repo, final JsonObject users, final String pattern) {
         return Json.createObjectBuilder()
-            .add("include-patterns", incpatterns)
+            .add("includesPattern", pattern)
             .add("repositories", Json.createArrayBuilder().add(repo).build())
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/api/artifactory/GetPermissionSliceTest.java:141]: @olegmoz I'm not sure I like these changes as `include-patterns` is actually a list, would not it be better to revert changes here and add several patterns in one test method?

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz several suggestion, take a look please

File: src/test/java/com/artipie/api/artifactory/GetPermissionSliceTest.java (hunk 4 above)
Comment (olegmoz) [reply src/test/java/com/artipie/api/artifactory/GetPermissionSliceTest.java:141]: @olenagerasimova that's not about test - it happens that original API has only one path in `GET` response, see https://www.jfrog.com/confluence/display/rtf/artifactory+rest+api#ArtifactoryRESTAPI-GetPermissionTargetDetails and `application/vnd.org.jfrog.artifactory.security.PermissionTarget+json` format description here: https://www.jfrog.com/confluence/display/JFROG/Security+Configuration+JSON

So original implementation was wrong. I noticed it only updating the docs

**Comment by olegmoz** [comment]
@olenagerasimova thanks, updated the docs as you suggested and replied about protocol changes

File: src/test/java/com/artipie/api/artifactory/GetPermissionSliceTest.java (hunk 4 above)
Comment (olenagerasimova) [reply src/test/java/com/artipie/api/artifactory/GetPermissionSliceTest.java:141]: @olegmoz oh, I see, thanks

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks, good to merge

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#618` is not assigned, can't get performer

**Pull request merged**